pr_agent/servers/github.py

```diff
@@ -0,0 +1,10 @@
+##获取github数据
+class GithubData(object):
+    def __init__(self):
+        self.des = "dssdd"
```

Comment:
**Suggestion:** Use a more descriptive name for the 'des' attribute in the 'GithubData' class.
```suggestion
class GithubData(object):
    def __init__(self):
        self.description = "dssdd"
```